### ISSUE TITLE
BIT * n invalid in TYPE with $NOPREFIX

### ISSUE THREAD

**Issue by SpriggsySpriggs** [QB64-Phoenix-Edition/QB64pe#179]
With $NOPREFIX enabled, BIT on its own works just fine for a member declaration in a TYPE. However, BIT * n causes an error saying "expected _BIT * number on line...".
When I change BIT to _BIT the error goes away.

```
Option Explicit
$NoPrefix

Type DCB
    As Unsigned Long DCBlength, BaudRate
    As Bit fBinary, fParity, fOutxCtsFlow, fOutxDsrFlow
    As _Bit * 2 fdtrControl
    As Bit fDsrSensitivity, fTXContinueOnXoff, fOutX, fInX, fErrorChar, fNull
    As _Bit * 2 fRtsControl
    As Bit fAbortOnError
    As _Bit * 17 fDummy2
    As Unsigned Integer wReserved, XonLim, XoffLim
    As Byte ByteSize, Parity, StopBits
    As Unsigned Byte XonChar, XoffChar, ErrorChar, EofChar, EvtChar
    As Unsigned Integer wReserved1
End Type
```

**Comment by SpriggsySpriggs** [comment]
It has been brought to my attention that BITs in user-defined TYPEs are horrible and I will be closing this issue.

**Comment by mkilgore** [comment]
I think I unintentionally fixed this in #202, it might be worth a test...
